specs/commandline/sign.md

```diff
@@ -28,27 +28,29 @@ Usage:
-       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
-  -u,  --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
-  -m,  --user-metadata strings    {key}={value} pairs that are added to the signature payload
+  -e,  --expiry duration            optional expiry that provides a "best by use" time for the artifact. The duration is specified in minutes(m) and/or hours(h). For example: 12h, 30m, 3h20m
+  -h,  --help                       help for sign
+  -k,  --key string                 signing key name, for a key previously added to notation's key list.
+  -p,  --password string            password for registry operations (default to $NOTATION_PASSWORD if not specified)
+       --plain-http                 registry access via plain HTTP
+       --plugin-config strings      {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
+       --signature-format string    signature envelope format, options: "jws", "cose" (default "jws")
+       --signature-manifest string  manifest type for signatures, options: "image", "artifact" (default "artifact")
+  -u,  --username string            username for registry operations (default to $NOTATION_USERNAME if not specified)
+  -m,  --user-metadata strings      {key}={value} pairs that are added to the signature payload
 ```
 
 ## Use OCI image manifest to store signatures
 
-By default, Notation uses [OCI artifact manifest][oci-artifact-manifest] to store signatures in registries. For backward compatibility, Notation supports using `OCI image manifest` to store signatures in registries that partially implement the [OCI Image specification v1.1][oci-image-spec]. Use flag `--image-spec v1.1-image` to force Notation to store the signatures using OCI image manifest.
+By default, Notation uses [OCI artifact manifest][oci-artifact-manifest] to store signatures in registries. For registries that don't support `OCI artifact` or [Referrers API][oci-referers-api] is not enabled in registries, users should use flag `--signature-manifest image` to force Notation to store the signatures using [OCI image manifest][oci-image-spec].
 
-Registries MAY not implement or enable the `Referrers API`, which is used by clients to fetch referrers. In the context of Notation, the referrers are signatures. Notation follows the fallback procedure defined in [OCI distribution spec][oci-backward-compatibility] if `Referrers API` is unavailable.
+Note that there is no deterministic way to determine whether a registry supports `OCI artifact` or not. The following response status contained in error messages MAY indicate that the registry doesn't support `OCI artifact`.
+
+- Response status `400 BAD Request` with error code `MANIFEST_INVALID` or `UNSUPPORTED`
```

Comment:
Instead of this should we say notation uses both availability of referees API and inability to push artifact manifest to determine support of artifact manifest?

Reply:
@priteshbandi it was mentioned in previous paragraph, copied below 

"For registries that don't support `OCI artifact` or [Referrers API][oci-referers-api] is not enabled, users SHOULD use flag `--signature-manifest image` to force Notation to store the signatures using [OCI image manifest][oci-image-spec]."

Is this OK?